src/node_contextify.cc

```diff
@@ -269,7 +269,10 @@ class ContextifyContext {
       CHECK(!debug_context.IsEmpty());
     }
     Environment* env = Environment::GetCurrent(args);
-    ScopedEnvironment env_scope(debug_context, env);
+    const int index = Environment::kContextEmbedderDataIndex;
+    if (!debug_context->GetAlignedPointerFromEmbedderData(index)) {
+      ScopedEnvironment env_scope(debug_context, env);
```

Comment:
Thanks for the heads up. I have not used structs with this pattern before and was unaware of the semantics.